tensorboard/plugins/scalar/scalars_plugin_test.py

```diff
@@ -68,6 +70,51 @@ def set_up_with_runs(self, run_names):
     context = base_plugin.TBContext(logdir=self.logdir, multiplexer=multiplexer)
     self.plugin = scalars_plugin.ScalarsPlugin(context)
 
+  def set_up_db(self):
+    self.db_path = os.path.join(self.get_temp_dir(), 'db.db')
+    self.db_uri = 'sqlite:' + self.db_path
+    db_module, db_connection_provider = application.get_database_info(
+        self.db_uri)
+    context = base_plugin.TBContext(
+        db_module=db_module,
+        db_connection_provider=db_connection_provider,
+        db_uri=self.db_uri)
+    self.core_plugin = core_plugin.CorePlugin(context)
+    self.plugin = scalars_plugin.ScalarsPlugin(context)
+
+  def generate_run_to_db(self, experiment_name, run_name):
+    tf.reset_default_graph()
+    global_step = tf.get_variable(
```

Comment:
I know I said this was best practice for simulating actual V2 summaries usage, but it's probably not necessary for tests - you can just set the step= parameter on .scalar() directly and feed it with a placeholder value, which is simpler.  Then there's no need to run a fake "train_op" or run tf.global_variables_initializer().

Reply:
Done! I totally forgot about how step argument is a tensor. It looks much cleaner now. Thank you 🙇